sidecars/scala/Dockerfile

```diff
@@ -20,23 +20,28 @@ RUN mkdir /projects ${HOME} && \
     done
 
 RUN apk --no-cache add openjdk11-jre-headless --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community && \
-    apk --no-cache add bash curl procps nss
+    apk --no-cache add bash curl git procps nss
 
 WORKDIR /projects
 
-ENV JAVA_HOME=/usr/lib/jvm/default-jvm/
+ENV JAVA_HOME=/usr/lib/jvm/default-jvm/ \
+    METALS_VERSION="0.9.5+12-c5a9a2ac-SNAPSHOT"
 
 RUN mkdir -p $HOME/.cache && ln -s $HOME/.cache /root/.cache && \
     mkdir -p $HOME/.ivy2 && ln -s $HOME/.ivy2 /root/.ivy2 && \
     mkdir -p $HOME/.sbt && ln -s $HOME/.sbt /root/.sbt && \
-    curl -Ls https://raw.githubusercontent.com/paulp/sbt-extras/master/sbt > /usr/local/bin/sbt && \
-    chmod 0755 /usr/local/bin/sbt && \
-    curl -Ls https://raw.githubusercontent.com/lefou/millw/main/millw > /usr/local/bin/mill && \
-    chmod 0755 /usr/local/bin/mill && \
+    git clone https://github.com/sideeffffect/console-scala-simple.git && \
```

Comment:
[devfile sample](https://github.com/eclipse/che-devfile-registry/blob/06a6b1684bcfefd19e1945dc0da0f87dedddf3cf/devfiles/scala-sbt/devfile.yaml#L10) points to the same here, this repo could be moved to github.com/che-samples as "scala-demo".